File: app/src/main/java/com/applications/whazzup/photomapp/ui/screens/splash/SplashView.java
package com.applications.whazzup.photomapp.ui.screens.splash;

import android.content.Context;
import android.util.AttributeSet;

import com.applications.whazzup.photomapp.di.DaggerService;
import com.applications.whazzup.photomapp.mvp.views.AbstractView;
import com.applications.whazzup.photomapp.mvp.views.ISplashView;


public class SplashView extends AbstractView<SplashScreen.SplashPresenter> implements ISplashView {


    public SplashView(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    @Override
    public boolean viewOnBackPressed() {
        return false;
    }

    @Override
    protected void initDagger(Context context) {
        DaggerService.<SplashScreen.SplashComponent>getDaggerComponent(context).inject(this);
    }

    @Override
    public void showLoad() {

    }

    @Override
    public void hideLoad() {

    }
}

File: app/src/main/java/com/applications/whazzup/photomapp/ui/screens/splash/SplashView.kt
package com.applications.whazzup.photomapp.ui.screens.splash

import android.content.Context
import android.util.AttributeSet

import com.applications.whazzup.photomapp.di.DaggerService
import com.applications.whazzup.photomapp.mvp.views.AbstractView
import com.applications.whazzup.photomapp.mvp.views.ISplashView


class SplashView(context: Context, attrs: AttributeSet) : AbstractView<SplashScreen.SplashPresenter>(context, attrs), ISplashView {

    override fun viewOnBackPressed(): Boolean {
        return false

    }

    override fun initDagger(context: Context) {
        DaggerService.getDaggerComponent<SplashScreen.SplashComponent>(context).inject(this)
    }

    override fun showLoad() {

    }

    override fun hideLoad() {

    }
}
